### PR TITLE
Renaming project from oci to ovirt_oci

### PR DESCRIPTION
**Pull request by bell-lev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# oci
+# Ovirt-CI
```
Comment (nirs) [inline README.md]: oVirt-CI?

File: ovirt_ci/config.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def config_parser():
     return cfg
 
 
-def load(path=os.path.expanduser("~/.config/oci.conf")):
+def load(path=os.path.expanduser("~/.config/ovirt_ci.conf")):
```
Comment (nirs) [inline ovirt_ci/config.py]: ~/.config/ovirt-ci.conf

File: test/config_test.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+from ovirt_ci import config
 
 
 def test_load_minimal(tmpdir):
     cfg_data = """[jenkins]
 user_id = fred
 api_token = 12345
 """
-    cfg_file = tmpdir.join("oci.conf")
+    cfg_file = tmpdir.join("ovirt_ci.conf")
```
Comment (nirs) [inline test/config_test.py]: ovirt-ci.conf

(same for all other conf file names in this file)

**Review by nirs — CHANGES_REQUESTED** [review]
Thanks Bell! sorry for the long delay.

**Comment by nirs** [comment]
@bellle do you plan to complete this?

**Comment by bell-lev** [comment]
@nirs , sorry for the delay - will do in the following days.

File: README.md (hunk 1 above)
Comment (bell-lev) [reply README.md]: Done

File: ovirt_ci/config.py (hunk 2 above)
Comment (bell-lev) [reply ovirt_ci/config.py]: Done

File: test/config_test.py (hunk 3 above)
Comment (bell-lev) [reply test/config_test.py]: Done

**Review by nirs — APPROVED** [review]
Thanks @bellle. We need a rebase but I can handle it.

**Comment by nirs** [comment]
Merged as 2372c7a775a9a02844d39dd65248ffde01223a13

**Pull request closed without merging**